Formula/arangodb.rb

```diff
@@ -16,14 +18,37 @@ class Arangodb < Formula
   depends_on :macos => :yosemite
   depends_on "openssl"
 
+  # see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87665
+  fails_with :gcc do
+    build 730
+    cause "Generates incorrect code"
+  end
+
   fails_with :gcc do
     build 820
     cause "Generates incorrect code"
   end
 
+  # the ArangoStarter is in a separate github repository;
+  # it is used to easily start single server and clusters
+  # with a unified CLI
+  resource "starter" do
+    url "https://github.com/arangodb-helper/arangodb/archive/0.14.0.tar.gz"
+    sha256 "43f9c6ecd492541c4616df523513898ad1512bec7baa245852ee4c401b786f2c"
+  end
+
   def install
     ENV.cxx11
 
+    resource("starter").stage do
+      ENV.append "GOPATH", Dir.pwd + "/.gobuild"
+      system "make", "deps"
+      system "go", "build", "-ldflags", "-X main.projectVersion=0.13.10 -X main.projectBuild=38953b049a38c844ffdbf667e55e60c6cfa22e70",
```

Comment:
There is a mismatch here: `0.13.10` when above it's 0.14.0 (and the checksum is wrong as well).

Reply:
@fxcoudert sorry, I missed that. Has been fixed now.